aea/crypto/base.py

```diff
@@ -89,14 +89,14 @@ def sign_transaction(self, transaction: Any) -> Any:
     @abstractmethod
     def recover_message(
         self, message: bytes, signature: str, is_deprecated_mode: bool = False
-    ) -> Address:
+    ) -> Tuple[Address, ...]:
```

Comment:
@marcofavorito a necessary breaking change (though not used anywhere in packages). Due to the properties of elliptic curves multiple addresses can match

Reply:
ok let's keep it in mind!